src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDurationMapping.java

```diff
@@ -0,0 +1,95 @@
+package uk.gov.hmcts.reform.sscs.helper.mapping;
+
+import uk.gov.hmcts.reform.sscs.ccd.domain.*;
+import uk.gov.hmcts.reform.sscs.service.holder.ReferenceDataServiceHolder;
+
+import java.util.Collections;
+import java.util.List;
+
+import static java.util.Objects.isNull;
+import static java.util.Objects.nonNull;
+import static org.apache.commons.lang3.StringUtils.isNotBlank;
+import static uk.gov.hmcts.reform.sscs.ccd.domain.YesNo.isYes;
+import static uk.gov.hmcts.reform.sscs.helper.mapping.HearingsCaseMapping.isInterpreterRequired;
+import static uk.gov.hmcts.reform.sscs.helper.mapping.HearingsDetailsMapping.getElementsDisputed;
+
+
+public final class HearingsDurationMapping {
+    public static final int DURATION_SESSIONS_MULTIPLIER = 165;
+    public static final int DURATION_DEFAULT = 30;
+    public static final int MIN_HEARING_DURATION = 30;
+    public static final int MIN_HEARING_SESSION_DURATION = 1;
+    public static final String DURATION_TYPE_NON_STANDARD_TIME_SLOT = "nonStandardTimeSlot";
+    public static final String DURATION_TYPE_STANDARD_TIME_SLOT = "standardTimeSlot";
+    public static final String DURATION_UNITS_MINUTES = "minutes";
+    public static final String DURATION_UNITS_SESSIONS = "sessions";
```

Comment:
Can you remove the constants left behind on HearingsDetailsMapping?

Reply:
Done